waku/v2/utils/wakuenr.nim

```diff
@@ -34,21 +34,25 @@ type
     Filter = 2,
     Lightpush = 3,
 
-func toFieldPair(multiaddrs: seq[MultiAddress]): FieldPair =
-  ## Converts a seq of multiaddrs to a `multiaddrs` ENR
-  ## field pair according to https://rfc.vac.dev/spec/31/
-  
+func getRawField*(multiaddrs: seq[MultiAddress]): seq[byte] =
   var fieldRaw: seq[byte]
 
   for multiaddr in multiaddrs:
     let
       maRaw = multiaddr.data.buffer # binary encoded multiaddr
       maSize = maRaw.len.uint16.toBytes(Endianness.bigEndian) # size as Big Endian unsigned 16-bit integer
-    
+
     assert maSize.len == 2
 
     fieldRaw.add(concat(@maSize, maRaw))
-  
+
+  return fieldRaw
```

Comment:
Need the raw field so that we can pass in the appropriate type to discv5 protocol setup. (for additional multiaddresses)